content/hi/docs/contribute/localisation_hi.md

```diff
@@ -0,0 +1,71 @@
+---
+title: कुबेरनेटिस प्रलेखन के स्थानीयकरण के लिए एक गाइड
+content_type: concept
+---
+<!-- overview -->
+
+कुबेरनेटिस प्रलेखन के स्थानीयकरण के लिए एक गाइड
+
+<!-- body -->
+
+## टीम लक्ष्य प्रबंधन
+
+कुबेरनेटिस दस्तावेज़ स्थानीयकरण टीम [समुदाय की स्थानीयकरण मार्गदर्शिका](/docs/contribute/localization/#branching-strategy) के अनुसार स्थानीयकरण के लिए टीम के लक्ष्य और विकास शाखाओं का प्रबंधन करती है। यह खंड हिंदी भाषा टीम के टीम लक्ष्य प्रबंधन के लिए विशिष्ट सामग्री से संबंधित है।
+
+हिंदी भाषा टीम `main` शाखा से शाखित शाखा का उपयोग करती है। और विकास शाखा के नाम में निम्नलिखित संरचना है:
+`dev-1.22-hi.1`
+
```

Comment:
```suggestion
आम तौर पर, नवीनतम रिलीज शाखा से अपनी कार्यशील शाखा बनाएं। नवीनतम रिलीज़ शाखा का नाम समय के साथ बदलता है। उदाहरण के लिए, जब हम `dev-1.22-hi.2` को `main` में मिलाते हैं, तो नई `/hi` स्थानीयकरण शाखा का नाम `dev-1.22-hi.2` हो सकता है। और उदाहरण के लिए, जब हम अंततः Kubernetes `1.25` दस्तावेज़ीकरण का स्थानीयकरण शुरू करते हैं, तो नई `/hi` स्थानीयकरण शाखा का नाम `dev-1.25-hi.1` हो सकता है।
```

Reply:
Generally, create your working branch from the latest release branch. The name of the latest release branch changes over time. For example, after we merge `dev-1.22-hi.2` into `main`, the name of the new `/hi` localization branch might become `dev-1.22-hi.2`.  And for example, when we eventually begin localizing the Kubernetes `1.25` documentation, the name of the new `/hi` localization branch might become `dev-1.25-hi.1`.

Reply:
```suggestion
अपस्ट्रीम रिलीज़ शाखा से अपनी कार्यशील शाखा बनाने के लिए, निम्नलिखित सिंटैक्स का उपयोग करें:
``
$ git checkout -b <working_branch_name> upstream/<release_branch_name>
``
उदाहरण के लिए:
``
$ git checkout -b localize-api-glossary upstream/dev-1.22-hi.1
``
```
Add one backtick to the examples above.